### DISCUSSION TITLE
Support for custom css and custom js for Auto1111

### DISCUSSION THREAD

**Discussion by LEv145** [AbdBarho/stable-diffusion-webui-docker#529] [Ideas]
Auto1111 support custom css file: `user.css` for adding custom css on startup: https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/394ffa7b0a7fff3ec484bcd084e673a8b301ccc8/modules/ui_gradio_extensions.py#L48

And if you add the `javascript/user.js` file, then the js code from there will also be loaded along with `auto1111` (It may not have been planned, but it works)

**Comment by ssj17vegeta** [comment]
Strongly support the idea, since it would allow to hide LORAs located in sub-folders.
